modules/gitops-release-notes-1-19-0.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-19-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-19-0_{context}"]
+= Release notes for {gitops-title} 1.19.0
+
+{gitops-title} 1.19.0 is available on {OCP} 4.14, 4.16, 4.17, 4.18, and 4.19.
+
+[id="errata-updates-1-19-0_{context}"]
+== Errata updates
+
+RHEA-2025:XXXX - {gitops-title} 1.19.0 security update advisory::
+Issued: 2025-XX-NN
++
+The list of enhancements that are included in this release are documented in the following advisory:
++
+* link:https://access.redhat.com/errata/RHEA-2025:XXXX[RHEA-2025:XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-19-0_{context}"]
+== New features
+
+Enhanced control over image pull policies for Argo CD components::
+The {gitops-title} Operator now provides administrators with flexible, multi-level control over the `imagePullPolicy` for container images used by the Argo CD, GitOpsService, and RolloutsManager components. This enhancement enables organizations to define a consistent image pulling strategy across all managed instances. The {gitops-title} Operator evaluates image pull policy settings in the following order:
+* Instance-level policy: Defined in the Custom Resource (CR) using the `.spec.imagePullPolicy` field. This overrides all other values for that instance.  
+* Subscription-level policy: Configured using the `IMAGE_PULL_POLICY` environment variable in the {gitops-title}  Subscription.  
+* Default policy: If no other policy is set, Argo CD defaults to `IfNotPresent`.
++
+The {gitops-title} Operator supports the following standard Kubernetes `imagePullPolicy` values:
+* `Always`: Argo CD always pulls the image from the registry.
+* `IfNotPresent`: Argo CD pulls the image only if it is not already present on the cluster node.
+* `Never`: Argo CD never pulls the image and uses only the locally available image.
++
```

Comment:
I think there is a main documentation page for this feature as well. Could we add a reference to it here?